templaters/layouts/withHolyGrailLayout.js

```diff
@@ -36,163 +7,8 @@ const withHolyGrailLayout = ({
-        `
-                .replace(/\s\s+/g, " ")
-                .replace(/\/\*(?!\*\/)*\*\//g, ""),
-            },
-          ]}
-        />
-
-        <HolyGrail.Layout>
-          <Header gridArea={HolyGrail.GridArea.Header}>
-            <Box
-              sx={{
-                display: "flex",
-                alignItems: "center",
-                ".logo-text": {
-                  variant: "links.unstyled",
-                  color: "sglightblue",
-                  display: ["none", "inherit", "inherit"],
-                },
-              }}
-            >
-              <LogoImage
-                logoURL="/static/splitgraph_logo.svg"
-                sx={{ marginRight: "1ch", padding: 0.5 }}
-              />
-              <a
-                href="/"
-                aria-label="homepage"
-                title="Home"
-                className="logo-text"
-              >
-                Splitgraph
-              </a>
-            </Box>
-            <Box sx={{ color: "muted", display: "flex", alignItems: "center" }}>
-              {middleHeader}
-              <Link href="/blog">Blog</Link>
-              <Link
-                href="/docs"
-                className={renderDocsHeaderLink ? "" : "desktop-only"}
-              >
-                Docs
-              </Link>
-              <Link className="button-link desktop-only" href="/connect">
-                Connect
-              </Link>
-              <Link className="button-link" href="/explore">
-                Data
-              </Link>
-            </Box>
-          </Header>
-
-          <Sidebar
-            gridArea={HolyGrail.GridArea.Nav}
-            rootNode={contentTree}
-            matchActiveNode={matchActiveNode}
-            Link={Link}
-            activeNodeId={activeNodeId}
-            // 1 for depth 2, because it's zero based
-            maxInitialStackDepth={1}
-            initialDepth={activeNodeDepth}
-          />
-
-          <Box className="content-wrapper">
-            <MainContent>
-              <ContentHeader depth={activeNode.depth}>
-                {renderTitleHeading && <Heading>{meta.title}</Heading>}
-              </ContentHeader>
-              <ContentBody>
-                <MdxPage components={mdxComponents} />
-              </ContentBody>
-              <ContentFooter>
-                {renderInterPageNav ? (
-                  <InterPageNav
-                    Link={Link}
-                    up={activeNode.up}
-                    left={activeNode.left}
-                    right={activeNode.right}
-                  />
-                ) : null}
-              </ContentFooter>
-            </MainContent>
-            <Footer Link={Link} />
-          </Box>
-
-          <aside className="right-sidebar">
-            {renderTOC ? <TocMdx /> : null}
-          </aside>
-        </HolyGrail.Layout>
-      </BaseLayout>
-    );
+    return <div>DEPRECATED</div>;
```

Comment:
Can we remove the component instead of deprecating it?

Any consumer of this component will be broken anyway :smile: 

I like removing code, so I am pleased to see those 186 lines removed

Reply:
See the commit - we'll do this in another branch.